packages/pyright-internal/src/tests/samples/dataclassTransform2.py

```diff
@@ -23,6 +23,8 @@ def model_field(
     field_specifiers=(ModelField, model_field),
 )
 class ModelMeta(type):
+    not_a_field: str
```

Comment:
You might want to add a generic base class to the test to make sure it is properly handled.